packages/components/organisms/f-checkout/src/components/_tests/Address.test.js

```diff
@@ -30,7 +34,9 @@ const $v = {
 };
 
 describe('Address', () => {
-    const propsData = {};
+    const propsData = {
+        tenant: 'au'
```

Comment:
Should we be including `nz` as well?

Reply:
Nope. Only AU has a required state field.